lib/ansible/modules/network/cnos/cnos_portchannel.py

```diff
@@ -0,0 +1,515 @@
+        required: Yes
+        default: Null
+        choices: [aggregation-group, bfd, bridgeport, description, duplex, flowcontrol, ip, ipv6, lacp, lldp,
+        load-interval, mac, mac-address, mac-learn, microburst-detection, mtu, service, service-policy,
+        shutdown, snmp, spanning-tree, speed, storm-control, vlan, vrrp, port-aggregation]
+    interfaceArg2:
+        description:
+            - This is an overloaded Port Channel second argument. Usage of this argument can be found is the User Guide referenced above.
+        required: No
+        default: Null
+        choices: [aggregation-group number, access or mode or trunk, description, auto or full or half,
+        recieve or send, port-priority, suspend-individual, timeout,     receive or transmit or trap-notification,
+        tlv-select, Load interval delay in seconds, counter, Name for the MAC Access List, mac-address in HHHH.HHHH.HHHH format,
+        THRESHOLD  Value in unit of buffer cell, <64-9216>  MTU in bytes-<64-9216> for L2 packet,<576-9216> for
+        L3 IPv4 packet, <1280-9216> for L3 IPv6 packet, enter the instance id, input or output, copp-system-policy,
+        type, 1000  or  10000  or   40000 or   auto, broadcast or multicast or unicast, disable or enable or egress-only,
+        Virtual router identifier, destination-ip or destination-mac or destination-port or source-dest-ip or
+        source-dest-mac or source-dest-port or source-interface or source-ip or source-mac or source-port]
+    interfaceArg3:
+        description:
+            - This is an overloaded Port Channel third argument. Usage of this argument can be found is the User Guide referenced above.
+        required: No
+        default: Null
+        choices: [active or on or passive, on or off, LACP port priority, long or short, link-aggregation or
+        mac-phy-status or management-address or max-frame-size or port-description or port-protocol-vlan or
+        port-vlan or power-mdi or protocol-identity or system-capabilities or system-description or system-name
+        or vid-management or vlan-name, counter for load interval, policy input name, all or Copp class name to attach,
+        qos, queing, Enter the allowed traffic level, ipv6]
+    interfaceArg4:
+        description:
+            - This is an overloaded Port Channel fourth argument. Usage of this argument can be found is the User Guide referenced above.
+        required: No
+        default: Null
+        choices: [key-chain, key-id, keyed-md5 or keyed-sha1 or meticulous-keyed-md5 or meticulous-keyed-sha1 or simple, Interval value in milliseconds,
+         Destination IP (Both IPV4 and IPV6),in or out, MAC address, Time-out value in seconds, class-id, request, Specify the IPv4 address,
+         OSPF area ID as a decimal value, OSPF area ID in IP address format, anycast or secondary, ethernet, vlan, 
```

Comment:
CI failure due to PEP 8 issue:

```
2017-02-21 22:09:43 ERROR: PEP 8: lib/ansible/modules/network/cnos/cnos_portchannel.py:90:115: W291 trailing whitespace (current)
```

You can run the PEP 8 tests locally with `make pep8`.